lib/carrierwave/storage/fog.rb

```diff
@@ -1,6 +1,10 @@
 # encoding: utf-8
 
-require "fog" unless defined?(::Fog)
+begin
+  require "fog/core"
+rescue LoadError
+  require "fog" unless defined?(::Fog)
```

Comment:
Here, on the other hand, there was no `rescue LoadError` before. And this file is only required in `lib/carrierwave/storage.rb` if `Fog` is defined. So the rescue shouldn’t be necessary.